vm/interpreter/src/vm.rs

```diff
@@ -396,8 +402,9 @@ where
 
     /// Applies the state transition for a single message.
     /// Returns ApplyRet structure which contains the message receipt and some meta data.
-    pub fn apply_message(&mut self, msg: &ChainMessage) -> Result<ApplyRet, String> {
+    pub fn apply_message(&mut self, msg: &ChainMessage) -> Result<ApplyRet, anyhow::Error> {
         self.apply_message_fvm(msg)
+            .map_err(|e| anyhow::anyhow!("{}", e))
     }
```

Comment:
Would be neater to rename `apply_message_fvm` to `apply_message` and then untangle the error handling.

Reply:
Will do.